platform/event/src/main/java/org/motechproject/event/MotechEvent.java

```diff
@@ -15,14 +16,16 @@
  */
 public class MotechEvent implements Serializable {
     public static final String EVENT_TYPE_KEY_NAME = "eventType";
```

Comment:
What's event type? Is this for the scheduler? If its only for the scheduler, we should move this to scheduler module, the scheduler service interface for example.